surfactant/plugin/manager.py

```diff
@@ -96,6 +96,25 @@ def set_blocked_plugins(pm: pluggy.PluginManager) -> None:
         pm.set_blocked(plugin_name)
 
 
+def is_plugin_blocked(pm: pluggy.PluginManager, plugin_name: str) -> bool:
+    """
+    Check if a plugin is blocked in the plugin manager.
+
+    Args:
+        pm (pluggy.PluginManager): The plugin manager instance.
+        plugin_name (str): The name of the plugin to check.
+
+    Returns:
+        bool: True if the plugin is blocked, False otherwise.
+    """
+    if pm.is_blocked(plugin_name):
+        logger.warning(
+            f"The plugin '{plugin_name}' is blocked. To unblock run, 'surfactant plugin enable {plugin_name}'"
+        )
```

Comment:
I'd lean towards letting whatever code is calling this function decide whether or not it needs to print a warning message.